src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -333,17 +379,16 @@ async def main(): # pylint: disable=too-many-locals
             "--no-print-out-dir",
         ]
 
-        if "output_directory_flags" in litani_caps:
-            out_prefix = proof_root / "output"
-            out_symlink = out_prefix / "latest"
-            out_index = out_symlink / "html" / "index.html"
-            cmd.extend([
-                "--output-prefix", str(out_prefix),
-                "--output-symlink", str(out_symlink),
-            ])
-            print(
-                "\nFor your convenience, the output of this run will be symbolically linked to ",
-                out_index, "\n")
+        out_prefix = proof_root / "output"
```

Comment:
Again I don't see the need for this change...

Reply:
As I mentioned in the description of the PR, the check for the associated Litani capability can be removed considering all consumers of the starter-kit use a version of Litani that is >= v1.16.0 and this particular capability was introduced in Litani v1.10.0

Reply:
As mentioned above, I will be removing this particular change from the PR.